src/main/java/teammates/ui/controller/Action.java

```diff
@@ -672,4 +676,22 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
     protected void excludeStudentDetailsFromResponseParams() {
         regkey = null;
     }
+
+    protected void verifyAccessibleForInstructorToFeedbackResponseComment(
```

Comment:
@sujeet14108 I feel that yes, creating an abstract class with just one method should be fine, on the justification that:

1. This helps reduce code duplication, so if this method were to be changed in the future it would not require looking through several different files.
2. There may be other methods that are similar between the `InstructorFeedbackResponseComment*Action` classes (including Add and Load) that could be extracted out as well to further improve code reuse. However, looking at these classes, I do not see anything else that might be appropriate for reuse.

However, I could be wrong, so I will have to defer to @damithc for this.